app/lib/donations/controllers/DonorsChooseDonationController.js

```diff
@@ -119,29 +116,24 @@ DonorsChooseDonationController.prototype.start = function(request, response) {
 };
 
 /**
- * Finds a project. Also responsible for sending the
+ * Queries DonorsChoose API to find a project and sends the
  * project details back to the user.
  *
- * @param request
- *   Express Request object
- * @param response
- *   Express Response object
+ * @see https://data.donorschoose.org/docs/project-listing/json-requests/
+ *
+ * @param mobileNumber
+ *   User's mobile number
+ * @param config
+ *   Loaded donorschoose config document.
  */
-DonorsChooseDonationController.prototype.findProject = function(mobileNumber, configId) {
-
-  var config = app.getConfig(app.ConfigName.DONORSCHOOSE, configId);
-
-  // Subject code for all 'Math & Science' subjects.
-  var subjectFilter = 'subject4=-4'; 
-  // Search returns results ordered by urgency algorithm. 
-  var urgencySort = 'sortBy=0'; 
-  // Constrains results which fall within a specific 'costToComplete' value range. 
-  var costToCompleteRange = 'costToCompleteRange=' + DONATION_AMOUNT + '+TO+' + COST_TO_COMPLETE_UPPER_LIMIT; 
-  var projectsCreatedBy = 'olderThan=' + PROJECT_CREATION_CUTOFF_DATE;
-  // Maximum number of results to return. 
-  var maxNumberOfResults = '1';
-  var filterParams = subjectFilter + '&' + urgencySort + '&' + costToCompleteRange + '&' + projectsCreatedBy + '&';
-  var requestUrlString = donorsChooseProposalsQueryBaseURL + filterParams + 'APIKey=' + donorsChooseApiKey + '&max=' + maxNumberOfResults;
+DonorsChooseDonationController.prototype.findProject = function(mobileNumber, config) {
+  var requestUrlString = donorsChooseProposalsQueryBaseURL;
+  requestUrlString += '&subject4=-4'; 
+  requestUrlString += '&sortBy=2'; 
```

Comment:
Intentional change here to `sortBy=2`
> Cost shows projects with the lowest cost to complete at the top

From previous requests for `sortBy=0:`
> Urgency shows projects with higher urgency at the top

Refs https://data.donorschoose.org/docs/project-listing/json-requests/